classes/abstracts/ActionScheduler.php

```diff
@@ -181,21 +181,26 @@ public static function init( $plugin_file ) {
 	}
 
 	/**
-	 * Issue deprecated warning if an Action Scheduler function is called in the shutdown hook.
+	 * Check whether the AS data store has been initialized.
 	 *
 	 * @param string $function_name The name of the function being called.
+	 * @return bool
 	 */
-	public static function check_shutdown_hook( $function_name ) {
```

Comment:
This function is public and has been shipped previously. It needs to be deprecated not removed, as 3rd party code could be using it already.

Reply:
> This function is public and has been shipped previously so it needs to be deprecated

Done.